### ISSUE TITLE
Tzdata Flakiness can cause app to crash on boot

### ISSUE THREAD

**Issue by michaeljguarino** [bitwalker/timex#396]
Timex's dependency on tzdata can cause the application to crash on boot.  Specifically, we get errors like:

```
** (MatchError) no match of right hand side value: 
{:error, {:shutdown, {:failed_to_start_child, Tzdata.EtsHolder, {{:badmatch, {:ok, 302, [{"Date", "Tue, 21 Nov 2017 22:43:11 GMT"}, 
{"X-Frame-Options", "SAMEORIGIN"}, {"Content-Security-Policy", "upgrade-insecure-requests"}, {"Location", "https://data.iana.org/time-zones/tzdata-latest.tar.gz"}, {"Cache-Control", "public, s-maxage=600, max-age=3600"}, ...
```

This is probably a minor issue with that library, but we would prefer our application not to have a boot dependency on https://data.iana.org/time-zones/tzdata-latest.tar.gz to use this library (which we really like!).  Is there any way we could disable the tzdata dependency through configuration, perhaps by suppressing the runtime in mix.exs and telling Timex not to use it?

(Also, we have had additional issues with tzdata's behavior in distillery releases, because it doesn't properly resolve priv directories with respect to the release)

**Comment by tubedude** [comment]
You can disable Tzdata auto update:
https://github.com/lau/tzdata#automatic-data-updates

**Comment by michaeljguarino** [comment]
I think tzdata makes a query on boot as well that is unaffected by disabling auto updates.  We were having issues even after setting that config.

**Comment by Baishan** [comment]
Have the same issue. Would be great to have the option to not use tzdata at all  (we are only interested in durations).

**Comment by Baishan** [comment]
Realised theres an Time module included with elixir since 1.5 that can durations etc.
